Resources/ServerInfo/Guidebook/ServerRules/WizDenRules.xml

```diff
@@ -0,0 +1,6 @@
+﻿<Document>
+  # Server Rules
+  Wizard's Den servers have two different rulesets.
+  - Most servers use the [textlink="standard ruleset" link="StandardRuleset"].
+  - Medium Roleplay (MRP) servers use the [textlink="MRP ruleset" link="MRPRuleset"], which has MRP amendments.
+</Document>
```

Comment:
this isn't needed since we can just choose which one to show based on the server config